README.md

```diff
@@ -8,15 +8,17 @@ This project involves creating a responsive website for Pet Care company that fe
 
 ## Setup and Usage
 
-1. Clone the repository to your local machine.
-2. Run `docker-compose up -d` OR `docker compose up -d`.
+1. Clone the repository to your local machine with :
+- `git clone https://github.com/Alexandru-Bulai/website-project.git`
+- Open the terminal and go to the folder `cd website-project`
+- Run `npm install` to get all the modules and dependencies ready
+2. Run `docker-compose up -d && npx jest --testPathPattern="test/*" --coverage --watch; docker-compose down`.
```

Comment:
Please revisit the comment I made earlier. 

Instead of manually typing the command here and then changing it when there will be changes in the future, replace it with a simple instruction such as the example provided in the comment. Npm run dev.

And in that command you can put the entire docker and jest commands. And when you make changes to them the reader will stay the same as it should